build.gradle.kts

```diff
@@ -185,7 +185,7 @@ dependencies {
     "sqplugins"("org.sonarsource.kotlin:sonar-kotlin-plugin:2.8.0.1093@jar")
     "sqplugins"("org.sonarsource.slang:sonar-ruby-plugin:1.9.0.3429@jar")
     "sqplugins"("org.sonarsource.html:sonar-html-plugin:3.5.0.3009@jar")
-    "sqplugins"("org.sonarsource.sonarlint.omnisharp:sonarlint-omnisharp-plugin:1.0.1.40480@jar")
+    "sqplugins"("org.sonarsource.sonarlint.omnisharp:sonarlint-omnisharp-plugin:1.1.0.41801@jar")
```

Comment:
Have we released already 1.1? when?

Reply:
Each build will produce an artifact that can be consumed. It is not necessarily a release.